### ISSUE TITLE
flagging a comment scrolls to the top of the page

### ISSUE THREAD

**Issue by pushcx** [lobsters/lobsters#1134]
Click to flag a comment, click on a reason. The flag is recorded and the comment collapsed, but also the browser scrolls to the top of the page. 

Looking at `application.js.erb` I see:

```
  on('click', '.comment a.flagger', (event) => {
    event.preventDefault();
```

And my first guess for this is that the call to `preventDefault` was missing, so I don't know off the top of my head what's wrong and wanted to file the bug to track it.

Maybe @ymeynot45 wants to take this?

**Comment by ymeynot45** [comment]
My plate is full today, but I will take a look at it tomorrow.  It should
be a fun bug to squash.

**Comment by vincentrolfs** [comment]
I cannot reproduce this, is this still up to date?

**Comment by johnpash** [comment]
I've done a fix here #1153
